devicon.json

```
@@ -8107,6 +8107,41 @@
         "color": "#CC0000",
         "aliases": []
     },
+    {
+        "name": "railway",
+        "altnames": [],
+        "tags": [
+            "hosting",
+            "platform",
+            "deployment",
+            "git"
```

I'm not sure about this one. Can railway hos your git repo in a similar manner to GitHub, Gitlab, Bitbucket, etc..?
```suggestion
            "git"
```

---

In that case I was mistaken. I was thinking more about how each Railway service keeps a log of deployment attempts for github repos.

---

aha, until now "git" has only been used for tools that build on top of git or extend git in some way. For example: GitHub, GitLab, Bitbucket, GitKraken, GitLens, etc...

As I understand it Railway is more similar to Vercel, in that it can deploy a website based on a git repository. So it's more that it can "use" git than that it builds on top of git, if that makes sense.

It's all subjective though, we don't have any hard rules for tags, so this is just how I view it.
Hope that clears it up a bit :)

---

Ohh, and I would argue that tools like Intellij and other JetBrains IDE's shouldn't have the git tag either. Even though they extend git with diff tools, and other niceties. This is because the IDEs' main purposes are not to extend git or host git repos. It's more of an "extension" to the main product.